tests/cases/conformance/classes/staticIndexSignature/staticIndexSignature5.ts

```diff
@@ -0,0 +1,24 @@
+// @strict: true
+
+class B {
+    static readonly [s: string]: number;
+    static readonly [s: number]: 42 | 233
+}
+
+interface I {
+    static readonly [s: string]: number;
```

Comment:
Wait... What? Indexes don't have static sides (they describe a single shape) - shouldn't this be an error?